### ISSUE TITLE
 New Connector - NFTPerp DEX

### ISSUE THREAD

**Issue by warproxxx** [hummingbot/hummingbot#6727]
### Description

Create a connector to add support for NFTPerp on Hummingbot.

NFTPerp has a Javascript SDK and various example repos on github to reflect all its platform actions. They are linked below:

1) [NFTPerp SDK](https://github.com/nftperp/nftperp-sdk)
2) [Live Trader](https://github.com/nftperp/nftperp-maker/blob/main/liveTrader.js): A library to perform complex actions like creating and canceling bunch of limit orders 
3) [API Docs](https://nftperp.readme.io/reference/nftperp-v2-apis)
4) [Protocol Documentation](https://nftperp.gitbook.io/core-docs/)

NFTPerp is deployed in Arbitrum. The latest contract addresses can be found [here](https://api.nftperp.xyz/contracts). Join the NFTPerp discord channel [here](https://discord.com/invite/nftperp) or contact [me](https://t.me/th1r3een_13) or [0xJose](https://t.me/The0xJose) on Telegram if you have any question. We want to get this done as soon as possible, so if we do not find a developer in a few days, we will take care of this task internally. So DM us before starting

### Bounty

- Sponsor: NFTPerp
- Bounty amount: 4000 USDC
- Developer portion: 3800 USDC

**Comment by warproxxx** [comment]
Assigned to  @0xcodercrane

**Comment by 0xcodercrane** [comment]
Thank you! I will do my best

**Comment by fengtality** [comment]
@warproxxx You can handle this bounty privately with @0xcodercrane and utilize a New Connector Proposal to get the connector merged into the codebase. We don't charge any fee in that case.

If you want our help shepherding the process (which includes working with the dev to test/merge in the pull request and facilitating governance), note that Hummingbot Foundation charges a bounty 20% processing fee (or 10% if HBOT).

@nikspz is the Hummingbot Foundation point person for bounties.

**Comment by warproxxx** [comment]
@fengtality a PR for nftperp integration has been pushed here https://github.com/hummingbot/gateway/pull/263

Can we get it reviewed/approved?
